tests/frontend_selenium/utils/utils.py

```diff
@@ -1,7 +1,80 @@
 import random
 import string
 
+def get_seed():
+      seed = 'violin chair festival kiss double belt hen differ stamp boy else bag'
```

Comment:
mnemonics shouldn't be on the code, can we get it from a config file or environment variable so we can pass in the repo's secrets?